### PR TITLE
My schedule + worshop capacity warning 

### PR DESCRIPTION
**Pull request by sophieH29**

### CODE REVIEW

File: src/behaviors/my-schedule-behavior.html [hunk 1]
```diff
@@ -43,8 +45,21 @@
...
+          this.set('app.user.sessions.' + id, true);    
+          
+          // [temp fix] to trigger app.sessions observer and re-filter my-schedule
+          // @ozasadnyy, I hope you won't have heart attack
+          // DevFest is coming :)
+          this.set('app.user.sessions.000', null);          
+          HOVERBOARD.Elements.Template.$.toast.showMessage('{$ addedToSchedule $}');     
+
+          if (isWorkshop) {
+            HOVERBOARD.Elements.Template.$.infoDialog.headerText = '{$ workshopWarning.headerText $}';
```
Comment (ozasadnyy) [inline src/behaviors/my-schedule-behavior.html:57]: Can we invoke open with options? Like:
```
HOVERBOARD.Elements.Template.$.infoDialog.open({
headerText : '',
infoText: '',
ctaButtonText: ''
})
```
?

Comment (sophieH29) [reply src/behaviors/my-schedule-behavior.html:57]: No, we can't, Iron-overlay-behaviour does not have such options when open dialog

Comment (ozasadnyy) [reply src/behaviors/my-schedule-behavior.html:57]: But `info-dialog` it's our component so we can define such function

Comment (sophieH29) [reply src/behaviors/my-schedule-behavior.html:57]: We can, but is it any difference?

Comment (ozasadnyy) [reply src/behaviors/my-schedule-behavior.html:57]: In this case, we will be able to use it later. But you are right, we are going to burn this website after the event 🔥

**Review by ozasadnyy — APPROVED** [review]

**Pull request merged**